website/docs/r/s3_bucket_inventory.html.markdown

```diff
@@ -0,0 +1,127 @@
+---
+layout: "aws"
+page_title: "AWS: aws_s3_bucket_inventory"
+sidebar_current: "docs-aws-resource-s3-bucket-inventory"
```

Comment:
This resource documentation page needs a sidebar link added to `website/aws.erb` 👍